README.md

```diff
@@ -9,7 +9,7 @@ Duckcoin is a cryptocurrency that will have zero fees, be decentralized and easy
 - [x] Easy to mine
 - [x] Zero fees
 
-Duckcoin is a fully working cryptocurrency that uses PoW to give miners rewards. However, it is currently operating on a centralized server-client model, which will obviously get me canceled if I release this right now. 
+Duckcoin is a fully working cryptocurrency that uses PoW protocol, which is also a protocol used by Bitcoin and Eth Classic, to give miners rewards. However, it is currently operating on a centralized server-client model, which will obviously get me canceled if I release this right now. 
```

Comment:
I don't think this is info that is too relevant or necessary, but I would be open to adding it in.
The word "protocol" is a bit vague, could you make it clearer? Protocols are usually about communication, aren't they?